### PR TITLE
Add notebook showing how to setup an EMT job and use it with pyiron

### PR DESCRIPTION
**Pull request by jkitchin**
See https://github.com/pyiron/pyiron/issues/1446 for context.

### CODE REVIEW

**Review by liamhuber — COMMENTED** [review]
Hi @jkitchin, looks good! I found all the extra material between the cells easy to read, and the pyiron table at the end is a nice touch. The comments and docstrings retain some of my brutal honesty about the current hiccups in pyiron, but IMO that's ok to leave in 😂 

Please change `from ase.build import bulk` and future calls of `bulk` with the project-based call: `pr.atomistics.structure.bulk`. You convert to pyiron `Atoms` inside the job anyhow (nice touch, btw), and since it's a pyiron tutorial I'd like to get people used to creating structures pyironically. In this case we are just wrapping ASE's bulk call, but we have other structure creators (e.g. for GBs), so it's nice if people start to see where in pyiron to look when building new structures.

Optionally, we might consider putting the volume on the same footing as energy in the new class, i.e. in `run_static` add `self.output.volume = self.structure.get_volume()` right after setting the energy, and then later modifying the bit where we duplicate the result in HDF for Murn's sake to `h5out["volume"] = [self.output.volume]`. It's not strictly necessary, but I think it adds some nice symmetry and consistency to the output.

**Comment by jkitchin** [comment]
I think I made those changes in [e268ae1](https://github.com/pyiron/pyiron/pull/1447/commits/e268ae17e46b800f4f41c10de095b24c9c810776).

File: notebooks/emt-pyiron.ipynb [hunk 1]
```diff
@@ -0,0 +1,2634 @@
...
+    "        # The template job comes with `input` and `output` attributes that will  \n",
+    "        # handle all the (de)serialization for you\n",
+    "        self.input.atoms = pr.atomistics.structure.bulk('Cu')  # Or whatever default\n",
+    "        # self.output.energy = None  # Let's just populate the field for now\n",
+    "        \n",
+    "    @property\n",
+    "    def structure(self):\n",
+    "        \"\"\"\n",
+    "        'structure' is the canonical name for an `Atoms` object in pyiron.\n",
+    "        The Murnhaghan job is expecting(?) a `structure` attribute on its input jobs.\n",
```
Comment (pmrv) [inline notebooks/emt-pyiron.ipynb:71]:
```suggestion
    "        The Murnhaghan job is expecting a `structure` attribute on its input jobs.\n",
```

That's correct!

Comment (jkitchin) [reply notebooks/emt-pyiron.ipynb]: I changed this.

**Comment by liamhuber** [comment]
I can't figure out how to change my review status to "approve" here on mobile, but I do. I started the tests and as long as we don't have any surprises there, I'm happy to merge. Nice contribution!

**Comment by jkitchin** [comment]
No hurry for me.

**Comment by jkitchin** [comment]
I think I did what you requested.

**Comment by jan-janssen** [comment]
Everything looks alright, but there seems to be an issue with our conda environment. Somehow the dependencies are not resolved correctly, this is not related to this pull request still I would like to fix it before we merge this.

**Comment by jan-janssen** [comment]
@jkitchin Can you once merge with the main branch? We had some issues with the resolving of the dependencies, but those should be solved now.

**Pull request merged**